src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -226,65 +226,36 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugSendSpellFailCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugSendSpellFailCommand(ChatHandler* handler, SpellCastResult result, Optional<uint32> failArg1, Optional<uint32> failArg2)
     {
-        if (!*args)
-            return false;
-
-        char* result = strtok((char*)args, " ");
-        if (!result)
-            return false;
-
-        uint8 failNum = (uint8)atoi(result);
-        if (failNum == 0 && *result != '0')
-            return false;
-
-        char* fail1 = strtok(nullptr, " ");
-        uint8 failArg1 = fail1 ? (uint8)atoi(fail1) : 0;
-
-        char* fail2 = strtok(nullptr, " ");
-        uint8 failArg2 = fail2 ? (uint8)atoi(fail2) : 0;
-
         WorldPacket data(SMSG_CAST_FAILED, 5);
         data << uint8(0);
         data << uint32(133);
```

Comment:
any idea if this magic number is from any enum or other constant?

Reply:
That's a spell id.
https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?spell=133

Reply:
alright, then just add a comment or something